libkbfs/tlf_journal.go

```diff
@@ -1201,11 +1199,10 @@ func (j *tlfJournal) removeFlushedMDEntry(ctx context.Context,
 }
 
 func (j *tlfJournal) flushOneMDOp(
-	ctx context.Context, end MetadataRevision,
-	maxMDRevToFlush MetadataRevision) (flushed bool, err error) {
+	ctx context.Context, maxMDRevToFlush MetadataRevision) (
+	flushed bool, err error) {
 	if maxMDRevToFlush == MetadataRevisionUninitialized {
-		// Short-cut `getNextMDEntryToFlush`, which would otherwise read
-		// an MD from disk and sign it unnecessarily.
+		// Nothing to do.
```

Comment:
Oh wait, maybe I see it now.  The issue is the check in `mdJournal.getNextMDEntryToFlush()` before the sign I guess.  Ok.